frontend/src/pages/NurseDashboard.jsx

```
@@ -1,8 +1,12 @@
+import React from 'react';
+import useAuthStore from '../store/useAuthStore';
+
 import React, { useState, useEffect } from 'react';
```

Biome (2.1.2)

[error] Shouldn't redeclare 'React'. Consider to delete it or rename it.

'React' is defined here:


(lint/suspicious/noRedeclare)